### PR TITLE
[DOCFIX] combine the docs of running alluxio in a cluster into one

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: docs/en/Running-Alluxio-on-a-Cluster.md [hunk 1]
```diff
@@ -9,13 +9,20 @@ priority: 2
 * Table of Contents
 {:toc}
 
-## Download Alluxio
+## Running Alluxio with a Single Master
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-a-Cluster.md]: Can we start on `#` headers?

File: docs/en/Running-Alluxio-on-a-Cluster.md [hunk 2]
```diff
@@ -9,13 +9,20 @@ priority: 2
...
 
-First download the Alluxio tar file, and extract it.
+The simplest way to deploy Alluxio on a cluster is to use one master. However, this single master becomes
+the single point of failure (SPOF) in an Alluxio cluster: if that machine or process became unavailable, 
+the cluster as a whole would be unavailable. We highly recommend running Alluxio masters in 
+[High Availability](#running-alluxio-with-high-availability) in production.
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-a-Cluster.md]: High Availability Mode

File: docs/en/Running-Alluxio-on-a-Cluster.md [hunk 3]
```diff
@@ -54,3 +61,165 @@ log in the directory `alluxio/logs`, or run a sample program:
 
 **Note**: If you are using EC2, make sure the security group settings on the master node allows
  incoming connections on the alluxio web UI port.
+
+## Running Alluxio with High Availability
+
+High availability in Alluxio is based upon a multi-master approach, where multiple master processes
+are running in the system. One of these processes is elected the leader and is used by all workers 
+and clients as the primary point of contact. The other masters act as standby, and use the shared 
+journal to ensure that they maintain the same file system metadata as the leader, and can rapidly
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-a-Cluster.md]: Break "and can rapidly take over ..." to its own sentence

File: docs/en/Running-Alluxio-on-a-Cluster.md [hunk 4]
```diff
@@ -54,3 +61,165 @@ log in the directory `alluxio/logs`, or run a sample program:
...
+## Running Alluxio with High Availability
+
+High availability in Alluxio is based upon a multi-master approach, where multiple master processes
+are running in the system. One of these processes is elected the leader and is used by all workers 
+and clients as the primary point of contact. The other masters act as standby, and use the shared 
+journal to ensure that they maintain the same file system metadata as the leader, and can rapidly 
+take over in the event of the leader failing.
+
+If the leader fails, a new leader is automatically selected from the available standby masters and
+Alluxio proceeds as usual. Note that while the switchover to a standby master happens clients may
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-a-Cluster.md]: switchover -> failover to use consistent terminology

File: docs/en/Running-Alluxio-on-a-Cluster.md (hunk 1 above)
Comment (yupeng9) [reply docs/en/Running-Alluxio-on-a-Cluster.md]: I think we don't use `#` in any other docs

Comment (aaudiber) [reply docs/en/Running-Alluxio-on-a-Cluster.md]: I see, nvm then

File: docs/en/Running-Alluxio-on-a-Cluster.md [hunk 5]
```diff
@@ -54,3 +61,165 @@ log in the directory `alluxio/logs`, or run a sample program:
 
 **Note**: If you are using EC2, make sure the security group settings on the master node allows
  incoming connections on the alluxio web UI port.
+
+## Running Alluxio with High Availability
+
+High availability in Alluxio is based upon a multi-master approach, where multiple master processes
+are running in the system. One of these processes is elected the leader and is used by all workers 
+and clients as the primary point of contact. The other masters act as standby, and use the shared 
+journal to ensure that they maintain the same file system metadata as the leader. And it can rapidly
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-a-Cluster.md]: And it -> A standby master

**Pull request merged**